### PR TITLE
Update operator to 0.0.4

### PR DESCRIPTION
**Pull request by anishasthana**
One thing to note: I haven't yet tested with the latest images

### CODE REVIEW

File: controllers/defaults.go [hunk 1]
```diff
@@ -5,5 +5,5 @@ package controllers
 // ***********************
 
 const (
-	InstaScaleImage = "quay.io/project-codeflare/instascale-controller:v0.0.3"
+	InstaScaleImage = "quay.io/project-codeflare/instascale-controller:v0.0.4"
```
Comment (anishasthana) [inline controllers/defaults.go:8]: @astefanutti if we don't edit this file how else would we update the default?

Comment (astefanutti) [reply controllers/defaults.go:8]: It's now to be set in the Makefile: https://github.com/project-codeflare/codeflare-operator/blob/a71303b7d9a2a59eb8310dbbaa351e2b43af256c/Makefile#L13

Ideally, we'll eventually have the dependencies defined there in a central place.

Comment (astefanutti) [reply controllers/defaults.go:8]: And run `make defaults`, but it's done with most make targets.

File: config/internal/mcad/deployment.yaml.tmpl [hunk 2]
```diff
@@ -25,7 +25,7 @@ spec:
           envFrom:
             - configMapRef:
                 name: mcad-{{.Name}}-config
-          image: 'quay.io/project-codeflare/mcad-controller:release-v1.29.57'
+          image: 'quay.io/project-codeflare/mcad-controller:release-v1.31.0'
```
Comment (astefanutti) [inline config/internal/mcad/deployment.yaml.tmpl:28]: This has to be update here for the MCAD CRDs: 

https://github.com/project-codeflare/codeflare-operator/blob/a71303b7d9a2a59eb8310dbbaa351e2b43af256c/config/crd/mcad/kustomization.yaml#L2

File: controllers/defaults.go (hunk 1 above)
Comment (anishasthana) [reply controllers/defaults.go:8]: Got it!

File: config/internal/mcad/deployment.yaml.tmpl (hunk 2 above)
Comment (anishasthana) [reply config/internal/mcad/deployment.yaml.tmpl:28]: Updated. Good catch!

File: controllers/defaults.go (hunk 1 above)
Comment (anishasthana) [reply controllers/defaults.go:8]: Fixed.

**Comment by anishasthana** [comment]
I opened #128 for the test cases fix.

**Comment by anishasthana** [comment]
/retest

The openshift ci failure seems to be a flake on their end.

**Pull request merged**